### PR TITLE
added make format command

### PR DESCRIPTION
**Pull request by MrTheodor**
As #197 will definitely break all existing and not merged pull requests, perhaps better is to first introduce the command to format the code.

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -60,3 +60,14 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/ DESTINATION ${PYTHON_INSTDIR}/esp
     PATTERN "CMakeFiles" EXCLUDE)
 
 add_dependencies(_espressopp gitversion)
+
+find_program(CLANG_FORMAT "clang-format")
+if (CLANG_FORMAT)
+    add_custom_target(
+            format
+            COMMAND clang-format
```
Comment (junghans) [inline src/CMakeLists.txt]: Use `${CLANG_FORMAT}` to allow user overwrite.

**Comment by MrTheodor** [comment]
It seems to be something wrong with codecov

**Comment by junghans** [comment]
Let me retrigger!

**Comment by junghans** [comment]
Something is messed up, _500 Server Error_ for https://codecov.io/gh/espressopp/espressopp/pull/198

**Pull request merged**